cmd/globalAlignmentAnchor/globalAlignmentAnchor.go

```diff
@@ -68,7 +106,7 @@ func mafToMatch(in_maf string, species1 string, species2 string) {
 	mafRecords := maf.Read(in_maf) // read input maf
 
 	// open output files to write line-by-line and create variable for error
-	out_maf_filename := strings.Replace(in_maf, ".maf", ".filtered.maf", 1)
+	out_maf_filename := strings.Replace(in_maf, ".maf", ".filtered.maf", 1) // this means the in_maf path name should not contain .maf, otherwise the string renaming will be triggered in the pathname instead of in the filename
```

Comment:
Can we have the user supply this output name?
Instead of using string.Replace, could we check for .maf with strings.HasSuffix (error if not) and then trim the name back three characters and add on filtered.maf?  Then we wouldn't need to worry about what else might be in the name/path